components/style/properties/properties.mako.rs

```diff
@@ -3003,6 +3003,14 @@ impl ComputedValues {
         % endfor
         set
     }
+
+    /// Create a `TransitionPropertyIterator` for this styles transition properties.
+    pub fn transition_properties<'a>(
+        &'a self
+    ) -> animated_properties::TransitionPropertyIterator<'a>
+    {
```

Comment:
`{` should go to the previous line I think.

Reply:
rustfmt probably disagrees, since CI is passing.

Reply:
It seems like rustfmt might not be processing Mako templates? In any case, I've formatted this to look like the other methods in the file.

Reply:
Yeah, we don't rustfmt this file.